HotelBooking/HotelBooking.API/Program.cs

```diff
@@ -16,6 +19,8 @@
 builder.Services.AddBusinessLogicLayer();
 builder.Services.AddApi();
 
+builder.Services.AddScoped<IValidator<GuestDto>, GuestValidator>();
```

Comment:
Dodajmy to w w naszej dedykowanej extension method dla warstwy biznesowej tam gdzie mamy serwisy dodawane